grakn-graph/src/main/java/ai/grakn/graph/internal/ThingImpl.java

```diff
@@ -78,7 +78,7 @@
         }
 
         return vertex().graph().factory().buildConcept(typeEdge.get().target());
-    });
+    }, Cacheable.type());
```

Comment:
Ugh . . . no later. I agree it needs to be prettier but I am too bored/lazy . . . blazy? to make that change now :P